source/clear-linux/get-started/bootable-usb/bootable-usb.rst

```diff
@@ -3,12 +3,30 @@
 Create a bootable Clear Linux USB drive
 #######################################
 
-Instructions to create a |CLOSIA| USB drive vary depending on the system you are
-using. See below for step-by-step instructions catered to your system:
+Instructions to create a |CLOSIA| USB drive vary depending on your operating
+system.
+
+.. _download-usb-image:
+
+Download the latest Clear Linux image
+*************************************
+
+There are 2 types of |CL| images suitable for burning onto and running
+off a USB drive:
+
+   * Live image: :file:`clear-[version number]-live.img.xz`
+   * Installer image: :file:`clear-[version number]-installer.img.xz`
+
+Go to the Clear Linux `image`_ repository and download the desired type.
+
+With the appropriate image downloaded, choose the step-by-step instructions
+applicable to your system:
 
 .. toctree::
     :maxdepth: 1
 
     bootable-usb-linux
     bootable-usb-windows
     bootable-usb-mac
```

Comment:
This looks fine in RST, but when rendered it looks like this:

With the appropriate image downloaded, choose the step-by-step instructions applicable to your system:

Create a bootable USB drive on Linux
Create a bootable USB drive on Windows
Verify the integrity of the Clear Linux image
Uncompress the Clear Linux image
Burn the Clear Linux image onto a USB drive
Next steps
Create a bootable USB drive on macOS

The "verify", "uncompress", "burn", and "next steps" shouldn't be there.

Reply:
This issue was being caused by the incorrect heading underline on the origninal.